src/web/components/Nav/ExpandedMenu/ExpandedMenuToggle/VeggieBurger.tsx

```diff
@@ -11,7 +10,7 @@ const veggieBurger = ({
     showExpandedMenu: boolean;
 }) => css`
     background-color: ${brandAlt[400]};
-    color: ${palette.neutral[7]};
+    color: ${neutral[7]};
```

Comment:
Is this `text.primary`?

Reply:
This is an interesting one. Technically the veggie burger is a call to action, so I didn't want to use `text.primary`, which is intended for body copy or heading text. 

There is a call to action text colour that matches `neutral.7` (`brandAltText.ctaSecondary`), however the corresponding background colour (`brandAltBackground.ctaSecondary`) doesn't match the background colour of the veggie burger button, so I was reluctant to use it. This is why I went with globals for both background and text colours in this instance.